tests/functional/test_soft_round.py

```diff
@@ -0,0 +1,11 @@
+import torch
+
+from neuralcompression.functional import soft_round
+
+
+def test_soft_round():
+    x = torch.linspace(-2.0, 2.0, 50)
+
+    y = soft_round(x, alpha=1e-13)
+
+    torch.testing.assert_close(x, y)
```

Comment:
Since we have exact mathematical operations, we could have the test verify that we perform that math. Right now it would just be having the same code in two locations, but if we change the implementation of `soft_round_inverse` in the future (e.g., to C++) our tests will still catch bugs.